gmr/tests/test_mvn.py

```diff
@@ -75,13 +74,18 @@ def test_sample_confidence_region():
 
 def test_probability_density():
     """Test PDF of MVN."""
+    mean = np.array([0.0, 1.0, 2.0])
+    covariance = np.array([[ 0.5, -1.0,  0.0],
+                           [-1.0,  5.0, -0.5],
+                           [ 0.0, -0.5,  1.0]])
+
     random_state = check_random_state(0)
     mvn = MVN(mean, covariance, random_state=random_state)
 
     x = np.linspace(-100, 100, 201)
-    X = np.vstack(list(map(np.ravel, np.meshgrid(x, x)))).T
+    X = np.vstack(list(map(np.ravel, np.meshgrid(x, x, x)))).T
     p = mvn.to_probability_density(X)
-    approx_int = np.sum(p) * ((x[-1] - x[0]) / 201) ** 2
```

Comment:
I guess the width of the bins for numerical integration was calculated incorrectly in the test case before? Or how can we explain that the test didn't fail before this PR?